src/Scripting/Core/Script.cs

```diff
@@ -443,7 +443,7 @@ internal new Task<ScriptState<T>> ContinueAsync(ScriptState previousState, Cance
 
         private async Task<ScriptState<T>> RunSubmissionsAsync(ScriptExecutionState executionState, ImmutableArray<Func<object[], Task>> precedingExecutors, Func<object[], Task> currentExecutor, CancellationToken cancellationToken)
         {
-            var result = await executionState.RunSubmissionsAsync<T>(precedingExecutors, currentExecutor, cancellationToken).ConfigureAwait(continueOnCapturedContext: true);
+            var result = await executionState.RunSubmissionsAsync<T>(precedingExecutors, currentExecutor, cancellationToken).ConfigureAwait(continueOnCapturedContext: false);
```

Comment:
This change is not strictly necessary, but I can't imagine why we'd want to continue inline, and there's no comment.  It seemed safer to change this to ```false```.